game-service/src/main/java/com/lms/gameservice/controller/GameController.java

```diff
@@ -82,10 +82,15 @@ public ResponseEntity<String> joinGame(
         }
     }
 
-    @PostMapping("/{gameId}/round/{roundId}/process")
-    public ResponseEntity<String> processRound(@PathVariable Long gameId, @PathVariable Long roundId, @RequestBody List<Long> winningTeamIds) {
+    @PostMapping("/{gameId}/processRound")
+    public ResponseEntity<String> processRound(@PathVariable Long gameId) {
```

Comment:
Not need to verify logged in? who calls process round? will system call will it be a cron job not really sure